src/server/ReplicaService.d.ts

```diff
@@ -1,25 +1,11 @@
+import { ReplicaClassToken, ReplicationTypes } from "../util";
 import { Replica } from "../index";
 
-type ReplicaClassToken<T extends keyof Replicas> =
-	| symbol
-	| {
-			/**
-			 * @deprecated
-			 * @hidden
-			 */
-			__originalName: T;
-	  };
-type ReplicationSetting = "All" | Map<Player, true> | Player;
-
-/**
- * @server
- */
 export interface ReplicaService {
 	/**
-	 * A reference of players that have received initial data
-	 * \- having received initial data means having access to all replicas that are selectively replicated to that player.
+	 * A reference of players that have received initial data \- having received initial data means having access to all replicas that are selectively replicated to that player.
 	 */
-	ActivePlayers: ReadonlyMap<Player, true>;
+	readonly ActivePlayers: ReadonlyMap<Player, boolean>;
```

Comment:
This is a map of `<Player, true>` pairs, never false.
```suggestion
	readonly ActivePlayers: ReadonlyMap<Player, true>;
```